packages/experimental/src/rest/Resource.ts

```diff
@@ -126,7 +126,9 @@ export default abstract class Resource extends BaseResource {
     return this.memo('#delete', () =>
       endpoint.extend({
         fetch(this: RestEndpoint, params: any) {
-          return endpoint.fetch.call(this, params).then(() => params);
+          return endpoint.fetch
+            .call(this, params)
+            .then(res => (typeof res === 'object' ? res : params));
```

Comment:
this allows for `null` as well, is that intended?

Reply:
ah good catch